### ISSUE TITLE
Needs logout feature

### ISSUE THREAD

**Issue by SergeyTrusov** [dcarroll/sublime-lightning#26]
I can't get metadata from another org, because plugin try to login into existing org.

I found solution: force -logout, but it isn't unfriendly.

**Comment by dcarroll** [comment]
Logging in to a different org will change the user/session context.

Dave Carroll

**Comment by SergeyTrusov** [comment]
Yes, but if I am working into 2 org in same time I want get components from these org, in current case I can't get component in 2nd org, because login feature use previous org url.
If I close sublime and open new instance instance problem is reproduced.
I think this is major problem for users, that use more than one instance.

**Comment by dcarroll** [comment]
Once you are logged into both orgs, you can switch between them use force
active -u username@mail.com.

On Wed, Jan 27, 2016 at 10:59 AM, Sergey Trusov <notifications@github.com>
wrote:

> Reopened #26 <https://github.com/dcarroll/sublime-lightning/issues/26>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dcarroll/sublime-lightning/issues/26#event-528569790>.
>



-- 
Thanks,
Dave

**Comment by SergeyTrusov** [comment]
I can't login in both orgs in same time.

**Comment by dcarroll** [comment]
No, Sergey, you cannot.  But, when you log into one, then the other, the
oauth token is kept for both.  That way when you just switch which is
active, you don't need to login again until one or the other or both have
expired.

Maybe you could suggest a solution for how this might work better?

On Thu, Jan 28, 2016 at 7:36 AM, Sergey Trusov <notifications@github.com>
wrote:

> I can't login in both orgs in same time.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dcarroll/sublime-lightning/issues/26#issuecomment-176239430>
> .
>



-- 
Thanks,
Dave
